### PR TITLE
New DS core component: Autocomplete

### PR DESCRIPTION
**Pull request by vmilan**
# Description

Shortcut: https://app.shortcut.com/cartoteam/story/333187/core-autocomplete
[sc-333187]

Styles for the autocomplete component in `carto-theme` and documentation in storybook.

Screenshot 2024-01-25 at 16 19 46

## Type of change

- Feature

### CODE REVIEW

File: packages/react-ui/storybook/stories/molecules/ToggleButton.stories.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ const options = {
       url: 'https://www.figma.com/file/nmaoLeo69xBJCHm9nc6lEV/CARTO-Components-1.0?node-id=1534%3A36258'
     },
     status: {
-      type: 'readyToReview'
+      type: 'validated'
```
Comment (VictorVelarde) [inline packages/react-ui/storybook/stories/molecules/ToggleButton.stories.js:75]: Great work 💪🏻

**Comment by VictorVelarde** [comment]
Approach LGTM @vmilan  I played with Storybook and the only thing I found was this (Medium > Error, but I think it's a generic issue)

image

If the text is too long, it 'jumps' into a 2nd line, and it's not visually appealing to me. Is spec like that?

**Comment by vmilan** [comment]
> Approach LGTM @vmilan I played with Storybook and the only thing I found was this (Medium > Error, but I think it's a generic issue)
> 
> image
> 
> If the text is too long, it 'jumps' into a 2nd line, and it's not visually appealing to me. Is spec like that?

Good catch, there are no specs for this use case, I'll talk to Jose to find a solution. Thanks!

**Comment by VictorVelarde** [comment]
Fix LGTM, thx for the effort here Vero!

**Review by VictorVelarde — APPROVED** [review]
👍🏻

**Pull request merged**